### PR TITLE
Update crashplan to 4.8.4

### PR DESCRIPTION
**Pull request by zsprackett**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/crashplan.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
   version '4.8.4'
   sha256 'b2ffb640e1083ee95d5de93b627cbfbdbe53dae3dd59f5ae93723621a575991e'
 
-  url "https://download.crashplan.com/installs/mac/install/CrashPlan/CrashPlan_#{version}_Mac.dmg"
+  # download.code42.com was verified as official when first introduced to the cask
+  url "https://download.code42.com/installs/mac/install/CrashPlan/CrashPlan_#{version}_Mac.dmg"
```
Comment (vitorgalvao) [inline Casks/crashplan.rb]: Can’t find the download link. Where is it?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/crashplan.rb (hunk 1 above)
Comment (zsprackett) [reply Casks/crashplan.rb]: It's behind a paywall, however, if you visit the existing site you'll see that it's an alias for download.code42.com.

```
host download.crashplan.com
download.crashplan.com is an alias for download.code42.com.
download.code42.com has address 162.222.46.54
download.code42.com has address 162.222.46.76
download.code42.com has address 162.222.45.129
download.code42.com has address 162.222.45.130
```

Curl won't connect to the current hostname due to certificate mismatch:

```
curl -vvI https://download.crashplan.com
* Rebuilt URL to: https://download.crashplan.com/
*   Trying 162.222.45.130...
* TCP_NODELAY set
* Connected to download.crashplan.com (162.222.45.130) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384
* ALPN, server accepted to use http/1.1
* Server certificate:
*  subject: C=US; ST=Minnesota; L=Minneapolis; O=Code 42 Software, Inc.; CN=*.code42.com
*  start date: Jul  7 00:00:00 2017 GMT
*  expire date: Sep  5 23:59:59 2019 GMT
*  subjectAltName does not match download.crashplan.com
* SSL: no alternative certificate subject name matches target host name 'download.crashplan.com'
* stopped the pause stream!
* Closing connection 0
* TLSv1.2 (OUT), TLS alert, Client hello (1):
curl: (51) SSL: no alternative certificate subject name matches target host name 'download.crashplan.com'
```

Things work properly with the hostname update:
```
curl -vvI https://download.code42.com
* Rebuilt URL to: https://download.code42.com/
*   Trying 162.222.46.54...
* TCP_NODELAY set
* Connected to download.code42.com (162.222.46.54) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
* TLSv1.2 (IN), TLS handshake, Server finished (14):
* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
* TLSv1.2 (OUT), TLS change cipher, Client hello (1):
* TLSv1.2 (OUT), TLS handshake, Finished (20):
* TLSv1.2 (IN), TLS change cipher, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Finished (20):
* SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384
* ALPN, server accepted to use http/1.1
* Server certificate:
*  subject: C=US; ST=Minnesota; L=Minneapolis; O=Code 42 Software, Inc.; CN=*.code42.com
*  start date: Jul  7 00:00:00 2017 GMT
*  expire date: Sep  5 23:59:59 2019 GMT
*  subjectAltName: host "download.code42.com" matched cert's "*.code42.com"
*  issuer: C=US; O=thawte, Inc.; CN=thawte SSL CA - G2
*  SSL certificate verify ok.
> HEAD / HTTP/1.1
> Host: download.code42.com
> User-Agent: curl/7.54.0
> Accept: */*
>
< HTTP/1.1 200 OK
HTTP/1.1 200 OK
< Server: nginx
Server: nginx
< Date: Tue, 28 Aug 2018 18:00:40 GMT
Date: Tue, 28 Aug 2018 18:00:40 GMT
< Content-Type: text/html; charset=utf-8
Content-Type: text/html; charset=utf-8
< Content-Length: 237
Content-Length: 237
< Last-Modified: Wed, 26 Aug 2009 22:04:18 GMT
Last-Modified: Wed, 26 Aug 2009 22:04:18 GMT
< Connection: keep-alive
Connection: keep-alive
< ETag: "4a95b162-ed"
ETag: "4a95b162-ed"
< X-Frame-Options: SAMEORIGIN
X-Frame-Options: SAMEORIGIN
< Accept-Ranges: bytes
Accept-Ranges: bytes

<
* Connection #0 to host download.code42.com left intact
```

Comment (vitorgalvao) [reply Casks/crashplan.rb]: @zsprackett That means this is now a [walled build](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) and as such we’ll have to [remove the cask](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#unofficial-vendorless-and-walled-builds) unless you find an alternative URL.

**Comment by zsprackett** [comment]
Found it:  https://www.crashplan.com/en-us/download/

File: Casks/crashplan.rb (hunk 1 above)
Comment (zsprackett) [reply Casks/crashplan.rb]: Found it using google:  https://www.crashplan.com/en-us/download/

You can see it redirects to the code42 url.

**Pull request merged**